### DISCUSSION TITLE
Styling filter for table

### DISCUSSION THREAD

**Discussion by martingrosche** [bryanmylee/svelte-headless-table#86] [Q&A] [answered]
Is it possible to change the default style of the filter input?

```
const table = createTable(readable(articles), {
  ...
  colFilter: addColumnFilters(),
  ..
});
```

**Comment by bryanmylee** [comment, marked as answer]
This package doesn't provide any styling. You will have to provide the UI for the column filters yourself.

You can refer to the example in the page about the filters [here](https://svelte-headless-table.bryanmylee.com/docs/plugins/add-column-filters).

More specifically, you can view the source code for the example in this [REPL](https://svelte.dev/repl/e5da30febf484b39b7a0c2c4030ce5c9?version=3.48.0).
